packages/nextjs/hooks/scaffold-eth/useDeployedContractInfo.ts

```diff
@@ -38,7 +37,7 @@ export const useDeployedContractInfo = <TContractName extends ContractName>(cont
     };
 
     checkContractDeployment();
-  }, [isMounted, contractName, deployedContract, publicClient]);
+  }, [isMounted, contractName, deployedContract, publicClient, targetNetwork]);
```

Comment:
we could also instead do `targeNetwork.id`

Reply:
I think it doesn't even need to be in the dependency array, not sure how it got there :sweat: